_tools/libddwaf-updater/update.go

```diff
@@ -298,6 +310,14 @@ func (t target) embedSourceDirective() string {
 }
 
 func init() {
+
+	forceFlag = flag.Bool("force", false, "Force the download of assets even if the version is the same")
+	versionFlag = flag.String("version", "latest", "Force the download of assets for a specific version (by git tag), or 'latest' for the latest release")
+
+	if forceFlag == nil || versionFlag == nil {
+		panic("unexpected nil flag")
+	}
```

Comment:
Which means here you'd want to use `BoolVar` and `StringVar`:

```suggestion
	flag.BoolVar(&forceFlag, "force", false, "Force the download of assets even if the version is the same")
	flag.StringVar(&versionFlag, "version", "latest", "Force the download of assets for a specific version (by git tag), or 'latest' for the latest release")
```